craft_application/models/project.py

```diff
@@ -301,7 +315,7 @@ def _validate_part(part: dict[str, Any]) -> dict[str, Any]:
 
 
 class Project(base.CraftBaseModel):
-    """Craft Application project definition."""
+    """A craft project."""
 
     name: ProjectName
```

Comment:
This already has other information in it from its Annotated type: https://github.com/canonical/craft-application/blob/3bd339cf70d19d039292cd1c1961e953b45a4899/craft_application/models/constraints.py#L102

An application can override that information by assigning a `pydantic.Field` to `name`.